### PR TITLE
#109

### LINKED ISSUE #109: add actions in block in /block/:block
by calling getActions GetActionsByBlockRequest in the section below, like https://iotexscan.io/blocks/333a3bd051869880ecb7c55cd60ee224f09bc340d7f3a9bd05f480b34566e595

you can reuse getActionColumns or ActionTable

### PR DESCRIPTION
**Pull request by uu-z**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind feature
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #109 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: src/shared/block/action-table.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import { publicKeyToAddress } from "iotex-antenna/lib/crypto/crypto";
+// @ts-ignore
+import { t } from "onefx/lib/iso-i18n";
+import React from "react";
+import { Query } from "react-apollo";
+import { Action } from "../../api-gateway/resolvers/antenna-types";
+import { SpinPreloader } from "../common/spin-preloader";
+import { GET_ACTIONS_BY_BLK } from "../queries";
+
+export function getActionColumns(): Array<ColumnProps<Action>> {
```
Comment (puncsky) [inline src/shared/block/action-table.tsx]: please don't copy code but reuse iotex-explorer/src/shared/address-details/action-table.tsx

File: src/shared/block/action-table.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  return {
+    byBlk: {
+      blkHash,
+      start: start < 0 ? 0 : start,
+      count: PAGE_SIZE
+    }
+  };
+}
+
+export function ActionTable({ blkHash }: { blkHash: string }): JSX.Element {
```
Comment (puncsky) [inline src/shared/block/action-table.tsx]: can we refactor existing ActionTable to support blkHash query?

File: src/shared/queries.ts [hunk 3]
```diff
@@ -236,6 +236,44 @@ export const GET_ACTIONS_BY_INDEX = gql`
   }
 `;
 
+export const GET_ACTIONS_BY_BLK = gql`
+  query getActions($byBlk: GetActionsByBlockRequest) {
+    getActions(byBlk: $byBlk) {
+      actions {
```
Comment (puncsky) [inline src/shared/queries.ts]: please rebase

Comment (uu-z) [reply src/shared/queries.ts]: merge all GET_ACTIONS_XXX in just one. it's that right?

**Pull request closed without merging**